src/checkout/checkout-store-selector.ts

```diff
@@ -323,9 +323,12 @@ export function createCheckoutStoreSelectorFactory(): CheckoutStoreSelectorFacto
         getConsignments => clone(() => {
             const consignments = getConsignments();
 
-            if (consignments && consignments.length) {
-                return consignments[0].availableShippingOptions;
+            if (!consignments) {
+                return;
             }
+            const shippingConsignment = consignments.find(consignment => !consignment.selectedPickupOption);
```

Comment:
```suggestion
            const shippingConsignment = consignments?.find(consignment => !consignment.selectedPickupOption);
```

So you don't need the `if` above? 🍹